src/include/simeng/GenericPredictor.hh

```diff
@@ -30,7 +30,7 @@ class GenericPredictor : public BranchPredictor {
    * branch type, and a known target if not 0. Returns a branch direction and
    * branch target address. */
```

Comment:
*known branch offset

Not sure what the "if not 0" bit is in the comment. It would make some sense, but it is badly written still, if it had a default value of 0 in the func.

Reply:
Updated typo. Should I add a default value or just remove that part of the comment?

Reply:
Given that we check against the offset being 0 as a way to detect no known offset, should probably have the default value. Should be on the BP unit to mandate that, no the function caller